Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1011,6 +1012,13 @@ - (void)enterFullScreen:(CGSize)videoDimensions
 
         [CATransaction commit];
     }];
+
+#if PLATFORM(VISION) && ENABLE(QUICKLOOK_FULLSCREEN)
```

Comment:
`#if ENABLE(QUICKLOOK_FULLSCREEN)` is enough
(I know we haven't been 100% consistent)